src/Libraries/Microsoft.Extensions.Http.Resilience/Hedging/StandardHedgingHandlerBuilderExtensions.cs

```diff
@@ -25,13 +25,16 @@ public static class StandardHedgingHandlerBuilderExtensions
     /// <param name="builder">The pipeline builder.</param>
     /// <param name="section">The section that the options will bind against.</param>
     /// <returns>The value of <paramref name="builder"/>.</returns>
-    [DynamicDependency(DynamicallyAccessedMemberTypes.All, typeof(HttpStandardHedgingResilienceOptions))]
     public static IStandardHedgingHandlerBuilder Configure(this IStandardHedgingHandlerBuilder builder, IConfigurationSection section)
     {
         _ = Throw.IfNull(builder);
         _ = Throw.IfNull(section);
 
-        var options = Throw.IfNull(section.Get<HttpStandardHedgingResilienceOptions>());
+        if (!section.GetChildren().Any())
+        {
+            Throw.ArgumentNullException(nameof(section));
```

Comment:
```suggestion
            Throw.ArgumentException(nameof(section));
```

"section" is not in fact null...

Reply:
Are you OK with the (technically) breaking change here? The existing code throws an `ArgumentNullException` in this case, which is why I choose to preserve the behavior.

Reply:
This is the test that verifies this:

https://github.com/dotnet/extensions/blob/7db3a7edc35ae8008c75631da35c01467912cacf/test/Libraries/Microsoft.Extensions.Http.Resilience.Tests/Hedging/StandardHedgingTests.cs#L150-L151

Reply:
Yes, I think the code was previously broken in that respect.

Reply:
Devs continuously use "ArgumentNullException" for non-arguments...

Reply:
I think the Throws.IfNull pattern makes this worse in that respect, developers just call this for any null value instead of just for arguments being null.